x-pack/docs/en/security/authorization/built-in-roles.asciidoc

```diff
@@ -168,10 +168,12 @@ to remove or add repositories or to restore snapshots. It also does not enable
 to change index settings or to read or update data stream or index data.
 
 [[built-in-roles-superuser]] `superuser`::
-Grants full access to the cluster, including all indices and data. A user with
-the `superuser` role can also manage users and roles and
-<<run-as-privilege, impersonate>> any other user in the system. Due to the
-permissive nature of this role, take extra care when assigning it to a user.
+Grants full access to cluster management and data indices. This role also grants
```

Comment:
I'm not sure about the accuracy of this ("cluster management") when taking into consideration operator privileges?

Reply:
That's a good point. Operator privileges are disabled by default for self-managed, but enabled in Cloud.

I'm not opposed to mention, but operator privileges apply to low-level privileges in general — not just to this role — so I'm not sure if this is the right place. Should we add a note at the beginning of the page, so it will apply to any role?

Reply:
That's a tricky one.
Operator privileges aren't specific to builtin roles, so I wouldn't want to put too much detail on this page. I think we could do something like:
1. Add a new paragraph to https://www.elastic.co/guide/en/elasticsearch/reference/current/operator-privileges.html that is a little more explicit about ESS/ECE that says something like  
   > When running Elasticsearch on Elastic Cloud (or Elastic Cloud Enterprise) Operator Privileges are enabled, which means that some infrastructure management functionality is restricted and cannot be accessed by your administrative users. This protects your cluster from unintended infrastructure changes.
2. Add a small note at the top of the builtin users page that refers to the main operator privileges docs  
   > When running on Elastic Cloud, ["operator privileges"](https://www.elastic.co/guide/en/elasticsearch/reference/current/operator-privileges.html) are enabled, and some infrastructure management functionality is restricted even if a role would otherwise permit the user to perform that task.
3. Add a very small comment in `superuser` like 
   > Note: On Elastic Cloud, all standard users, including those with `superuser` are restricted from performing [operator-only](https://www.elastic.co/guide/en/elasticsearch/reference/current/operator-privileges.html) actions

Maybe that's too much, but I think it's manageable, and it explicit about the behaviour where it is most likely to be read.

It might be worth doing that in a separate PR though. It's a distinct piece of work and it would make it easier to iterate on the wording.

Reply:
I opened https://github.com/elastic/elasticsearch/issues/82462 to address additional changes, we can iterate there.